python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py

```diff
@@ -311,6 +317,17 @@ def __getitem__(self, item):
             self._caches[item] = self._build_once(item)
         return self._caches[item]
 
+    def get_program(self, item):
+        if not isinstance(item, FunctionSpec):
+            raise ValueError(
+                "Input item's type should be FunctionSpec, but received %s" %
+                type(item))
+        if item not in self._caches:
+            raise RuntimeError(
+                'Failed to find program for input item, please decorate input function by declarative.'
```

Comment:
please decorate the input function by "@declarative" may be better

Reply:
done, thx